### DISCUSSION TITLE
Add images to libraries

### DISCUSSION THREAD

**Discussion by garrettreynold445** [excalidraw/excalidraw#8270] [Q&A]
Hi All - We just installed Excalidraw on Vercel via github and its working correctly - we wanted  to add pngs or svgs to our personal libaries - we know that Excalidraw does not support adding images  in the public  library yet, and the converter isn’t working properly but the feedback we got was  was that we could "modify the code to allow image embedding." 

So we spent the last 1.5 days reviewing the  excalidraw codebase to see how  we could do this for our fork  and tried lots of options including  writing a python script to work as an external tool that converts svg to excalidraw - even thought the conversion is successful, excalidraw still can't accept it as a  library file. We tried other things like changing file extension types but no success and
 we have now run out of ideas. 
However, if anyone has already done this or has suggestions on how we can do it please let us know. We are react / python devs and if a solution is successful we will happily share the solution code with the community

thanks, Garrett

**Comment by Mrazator** [comment]
Hi, images in libraries are in our backlog, let's track it here https://github.com/excalidraw/excalidraw/issues/5034
